client/blocks/like-button/button.jsx

```diff
@@ -58,7 +58,10 @@ const LikeButton = React.createClass( {
 			'has-count': showLikeCount,
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Like', { context: 'verb: imperative'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/i18n-like-button-context)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `comment`

 Would it make sense to reuse one of the above?